common/components/AddressBookTable.scss

```diff
@@ -0,0 +1,85 @@
+@import 'common/sass/variables/spacing';
+@import 'common/sass/variables/colors';
+
+.AddressBookTable {
```

Comment:
The Address book is still lacking some support for smaller screens.

<img width="677" alt="screen shot 2018-05-01 at 3 42 22 pm" src="https://user-images.githubusercontent.com/13836920/39490354-9cfe32b8-4d56-11e8-8b9c-bbd5416b2631.png">

Reply:
Oh, that must be right before it collapses. Should be a quick fix.